CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ Thanks for your interest in Presto.  Our goal is to build a fast, scalable and r
 
 ## Getting Started
 
-Presto's [open issues are here](https://github.com/prestodb/presto/issues). Issues that would make a good first pull request for new contributors with the `beginner-task` tag. An easy way to get started helping the project is to *file an issue*. Issues can include bugs, new features, or documentation that looks outdated. For community support, [ask for help in Slack](https://join.slack.com/t/prestodb/shared_invite/enQtNTQ3NjU2MTYyNDA2LTYyOTg3MzUyMWE1YTI3Njc5YjgxZjNiYTgxODAzYjI5YWMwYWE0MTZjYWFhNGMwNjczYjI3N2JhM2ExMGJlMWM).
+Presto's [open issues are here](https://github.com/prestodb/presto/issues). Issues that would make a good first pull request for new contributors with the `good-first-issue` tag. An easy way to get started helping the project is to *file an issue*. Issues can include bugs, new features, or documentation that looks outdated. For community support, [ask for help in Slack](https://join.slack.com/t/prestodb/shared_invite/enQtNTQ3NjU2MTYyNDA2LTYyOTg3MzUyMWE1YTI3Njc5YjgxZjNiYTgxODAzYjI5YWMwYWE0MTZjYWFhNGMwNjczYjI3N2JhM2ExMGJlMWM).
```

Comment:
Maybe link to the issue: https://github.com/prestodb/presto/labels/good%20first%20issue

Also there are no dashes

Reply:
> Maybe link to the issue: [ good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue)
> 
> Also there are no dashes

I didn't think of linking to the tag! Done.